src/OSPSuite.Infrastructure.Import/Core/IColumnDescription.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Collections.Generic;
+
+namespace OSPSuite.Infrastructure.Import.Core
+{
+   public class ColumnDescription
+   {
+      public enum MeasurementLevel
+      {
+         NotSet,
+         Discrete,
+         Numeric
+      }
+
+      public ColumnDescription(int index)
+      {
+         Index = index;
+         Level = MeasurementLevel.NotSet;
+         ExistingValues = new List<string>();
+      }
+
+      //not sure we are going to be needing this kind of constructor
+      public ColumnDescription(int index, List<string> existingValues, MeasurementLevel columnDataType)
+      {
+         Index = index;
+         ExistingValues = existingValues;
+         Level = columnDataType;
+      }
+
+      public int Index { get; private set; }
+      public IList<string> ExistingValues { get; set; }
```

Comment:
HEre again encapsulation of values. Let's make sure we do not expose (in general, of course there are exceptions) remember as IList. They should be IReadonlyList<T> without get